internal/db/schema/migrations/postgres/2/01_credential.up.sql

```diff
@@ -0,0 +1,101 @@
+begin;
```

Comment:
Curious on what the procedure is for when we increment the migration base directory and when we don't?  I see you bumped it to 2 here

Reply:
The number is incremented for each release. I incremented it since this work is on a feature/long lived branch. It will need to be incremented again if we cut another release before this branch is merged into main (which is highly likely).